model/app.py

```diff
@@ -218,7 +218,7 @@ def key_from_model_info(wild_type_id, message):
     :param message: dict
     :return: str
     """
-    d = {k: message.get(k, []) for k in {GENOTYPE_CHANGES, MEDIUM, MEASUREMENTS}}
```

Comment:
let's keep this change, it shoudn't make a difference now, but will be beneficial once we start saving/restoring session changes